File: executioncontext/installationcontext/src/main/java/org/jetbrains/bsp/bazel/installationcontext/entities/InstallationContextDebuggerAddressEntity.java
package org.jetbrains.bsp.bazel.installationcontext.entities;

import com.google.common.net.HostAndPort;
import org.jetbrains.bsp.bazel.executioncontext.api.entries.ExecutionContextSingletonEntity;

public class InstallationContextDebuggerAddressEntity
    extends ExecutionContextSingletonEntity<HostAndPort> {

  public InstallationContextDebuggerAddressEntity(HostAndPort value) {
    super(value);
  }

  @Override
  public String toString() {
    return "InstallationContextDebuggerAddressEntity{" + "value=" + value + '}';
  }
}

File: executioncontext/installationcontext/src/main/kotlin/org/jetbrains/bsp/bazel/installationcontext/InstallationContextDebuggerAddressEntity.kt
package org.jetbrains.bsp.bazel.installationcontext

import org.jetbrains.bsp.bazel.executioncontext.api.ExecutionContextSingletonEntity

data class InstallationContextDebuggerAddressEntity(override val value: String) :
    ExecutionContextSingletonEntity<String>()
